pyproject.toml

```diff
@@ -29,7 +29,8 @@ open-autonomy = "==0.19.7"
 autonomy-dev = {version = "==0.2.153", extras = ["all"]}
 ccxt = ">=3.1.43"
 numpy = ">=1.26.0,<2"
-derive-client = ">=0.2.9"
+# derive-client = ">=0.2.9"
+derive-client = { path = "../../derive_client", develop = true }
```

Comment:
this needs to be set to the latest upstream version after it's release

Reply:
No it needs to reference the thirdparty as its a dev dependecy

Reply:
the prod build i.e. the agent should reference this i think?